hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -965,16 +969,30 @@ public void testDistributedTestDataSource() {
     assertEquals(1000, c);
   }
 
-  private static void prepareParquetDFSFiles(int numRecords) throws IOException {
-    String path = PARQUET_SOURCE_ROOT + "/1.parquet";
+  protected static void prepareParquetDFSFiles(int numRecords, String baseParquetPath) throws IOException {
+    prepareParquetDFSFiles(numRecords, baseParquetPath, "1.parquet");
```

Comment:
IMO, we should not hard code this parquet file. It will reduce the readability. We can extract it to be a constant.